vignettes/03-get-images-python.Rmd

```diff
@@ -1 +1,106 @@
-# Images Vignette
\ No newline at end of file
+---
+title: "Get Source Image Files"
+output: html_document
+---
+
+# Objective: To be able to demonstrate how to locate and retrieve RGB image files
+
+This vignette shows how to locate and retrieve image files associated with growing Season 6 
+from the University of Arizona's [Maricopa Agricultural Center](http://cals-mac.arizona.edu/) 
+using Python. The files are stored online on the data management system Clowder, 
+which is accessed using an API. We will be working with the image files generated during the 
+month of May by limiting the requests to that time period.
+
+After completing this vignette it should be possible to search for and retrieve other
+files through the use of the API.
+
+As an added bonus we've also included an exmple of how to retrieve the list of available 
+sensor names through the API. By using the sensor names returned, it's possible to retrieve
+other files containing the data the sensors have collected.
+
+**requirements** 
+* Python 3 
+* the terrautils library
+   * this can be installed from pypi by running `pip install terrautils` in the terminal
+* an API key to access these data
+
+## Locating the images
+
+To begin looking for files, a sensor name and site name are needed. We will be using 
+'RGB GeoTIFFs Datasets' as the sensor name and '' as the site name. Later in this
+vignette we show how to retrieve the list of available sensors.
+
+As mentioned in the overview, the url string will point to the API to use. In this case
+we'll be using "https://terraref.ncsa.illinois.edu/clowder/api" and the key will be the
+one you received in an email.
+
+```{python eval=FALSE}
+from terrautils.products import get_file_listing
+
+url = 'https://terraref.ncsa.illinois.edu/clowder/api'
+key = 'YOUR_KEY_GOES_HERE'
+sensor = 'RGB GeoTIFFs Datasets'
+sitename = ''
+files = get_file_listing(None, url, key, sensor, sitename,
+                   since='2018-05-01', until='2018-05-31')
```

Comment:
even after adding my API key, I get the following error:

> Traceback (most recent call last):
>   File "<stdin>", line 2, in <module>
>   File "/usr/local/lib/python3.7/site-packages/terrautils/products.py", line 157, in get_file_listing
>     sens = get_sensor(connection, host, key, sensor, sitename)
>   File "/usr/local/lib/python3.7/site-packages/terrautils/products.py", line 101, in get_sensor
>     r.raise_for_status()
>   File "/usr/local/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
>     raise HTTPError(http_error_msg, response=self)
> requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://terraref.ncsa.illinois.edu/clowder/apiapi/geostreams/streams?key=82127cf0-c76d-40dd-b313-58c5d47ae0e9&stream_name=RGB+GeoTIFFs+Datasets

Reply:
It looks like you have 'apiapi' in the URL. Is that a copy/paste error or is something not clear?